GitUI/AutoCompletion/CommitAutoCompleteProvider.cs

```diff
@@ -158,8 +158,16 @@ private static IEnumerable<string> ReadOrInitializeAutoCompleteRegexes()
         {
             if (file.IsTracked)
             {
-                GitCommands.Patches.Patch changes = await module.GetCurrentChangesAsync(file.Name, file.OldName, file.Staged == StagedStatus.Index, "-U1000000", noLocks: true)
-                .ConfigureAwait(false);
+                GitCommands.Patches.Patch changes = null;
```

Comment:
Can we do this? (Can't remember if we enabled annotations)
```suggestion
                GitCommands.Patches.Patch? changes = null;
```

Reply:
changed